BackofficeBundle/Resources/views/BackOffice/Underscore/Include/superboxView_crop._tpl.twig

```diff
@@ -8,23 +8,23 @@
 
     <!-- CROPPED VERSION IN USAGE -->
     <div class="form-group form-horizontal">
-        <label class="control-label col-lg-3">{{ 'open_orchestra_backoffice.media.current_preview'|trans }}</label>
+        <label class="control-label col-lg-3">{{ 'open_orchestra_mediaadmin.media.current_preview'|trans }}</label>
```

Comment:
Can you change it from open_orchestra_mediaadmin to open_orchestra_media_admin ?

It will be more consistent with the service declaration